Python/pywarpx/picmi.py

```diff
@@ -1988,20 +1988,19 @@ def laser_antenna_initialize_inputs(self, laser):
         self.normal_vector = laser.laser.direction  # The plane normal direction
         if isinstance(laser, GaussianLaser):
             # Focal distance from the antenna (in meters)
-            laser.laser.profile_focal_distance = np.sqrt(
-                (laser.focal_position[0] - self.position[0]) ** 2
-                + (laser.focal_position[1] - self.position[1]) ** 2
-                + (laser.focal_position[2] - self.position[2]) ** 2
+            laser.laser.profile_focal_distance = (
+                (laser.focal_position[0] - self.position[0]) * self.normal_vector[0]
+                + (laser.focal_position[1] - self.position[1]) * self.normal_vector[1]
+                + (laser.focal_position[2] - self.position[2]) * self.normal_vector[2]
```

Comment:
I quickly checked, and I don't think we ensure `self.normal_vector` to also be a unit vector. That might be worth adding as a precaution since a non-unit normal vector will cause problems here. Adding something like
```
# ensure the normal vector is a unit vector
self.normal_vector /= np.linalg.norm(self.normal_vector)
```
at line 1989 above would be good.

Also, it is unfortunate that the WarpX parameter is profile_focal_**distance** since a negative distance is kinda strange but it's not really worth making a breaking change for input files to update that. It would be worth changing the comment on line 1990 to say something like `displacement` instead of distance, but that is just nit-picking from my side.